shared/build.gradle.kts

```
@@ -43,26 +47,98 @@ kotlin {
             dependencies {
                 // Koin
                 implementation(Dependencies.Koin.CORE)
+
+                // Ktor
+                with(Dependencies.Ktor) {
+                    implementation(CORE)
+                    implementation(SERIALIZATION)
+                    implementation(LOGGING)
+                    implementation(CIO)
+                    implementation(CONTENT_NEGOTIATION)
+                    implementation(JSON)
+                    implementation(AUTH)
+                }
+                implementation(Dependencies.Logging.NAPIER)
             }
         }
         val commonTest by getting {
             dependencies {
                 implementation(kotlin("test"))
+                with(Dependencies.Test) {
+                    implementation(COROUTINES)
+                    implementation(KOTEST_ASSERTIONS)
+                    implementation(KOTLINX_RESOURCES)
+                    implementation(MOCKATIVE)
+                    implementation(TURBINE)
+                }
             }
         }
         val androidMain by getting {
             dependencies {
                 implementation(Dependencies.Koin.ANDROID)
                 implementation(Dependencies.Koin.COMPOSE)
+                implementation(Dependencies.Ktor.ANDROID)
+            }
+        }
+
+        val iosX64Main by getting
+        val iosArm64Main by getting
+        val iosSimulatorArm64Main by getting
+        val iosMain by getting {
+            dependsOn(commonMain)
+            iosX64Main.dependsOn(this)
+            iosArm64Main.dependsOn(this)
+            iosSimulatorArm64Main.dependsOn(this)
+            dependencies {
+                implementation(Dependencies.Ktor.IOS)
             }
         }
     }
 }
 
+// https://github.com/mockative/mockative#installation-for-multiplatform-projects
+dependencies {
+    configurations
+        .filter { it.name.startsWith("ksp") && it.name.contains("Test") }
+        .forEach {
+            add(it.name, Dependencies.Test.MOCKATIVE_PROCESSOR)
+        }
+}
+
+ksp {
```

please a note for reason

```suggestion
// https://github.com/mockative/mockative#implicit-stubbing-of-functions-returning-unit
ksp {
```